main.py

```diff
@@ -1,12 +1,199 @@
+    
+    return choice
+
+def run_cli(play=True):
+
+    #initialize retro_video_store
+    retro_video_store = Video_Store(url="http://127.0.0.1:5000")
+    
+    # print choices
+    options = list_options()
+
+    while play==True:
+
+        # get input and validate
+        choice = make_choice(options, retro_video_store)
+
+        retro_video_store.print_selected()
+
+        if choice=='1':
+            print_hashtags()
+            print("Here are all of the customers we have in our database.")
+            for customer in retro_video_store.list_of_customers():
+                print(customer)
+
+        elif choice=='2':
+            id=input("What is the id of the customer you want information about? ")
+            if id.isnumeric():
+                id=int(id)
+                print("Here is this customers information:", retro_video_store.get_customer(id=id))
+            else:
+                print("Please enter the number id.")
+
+        elif choice=="3":
+            name=input("What is the name of this new customer? ")
+            postal_code=input("What is their postal code? ")
+            phone=input("What's their phone number that they'll answer the most? ")
+            response = retro_video_store.create_customer(name=name, postal_code=postal_code, phone=phone)
+            print("New Customer id:", response["id"])
+
+        elif choice=='4':
+            print(f"Time to update someone's information!: {retro_video_store.selected_customer}")
+            name=input("What is their new name? ")
+            postal_code=input("What is their new postal code? ")
+            phone=input("What is their new phone number (if they changed it)? ")
+            response = retro_video_store.update_customer(id, name=name, postal_code=postal_code, phone=phone)
+
+            print_hashtags()
+            print("Updated customer name:", response["name"])
+
+        elif choice=='5':
+            customer_to_delete = input("What is the id of the customer you wish to delete? ")
+            if customer_to_delete.isnumeric():
+                customer_to_delete = int(customer_to_delete)
+                retro_video_store.delete_customer(customer_to_delete)
+
+            print_hashtags()
+            print("That customer has been deleted.")
+
+            print_hashtags()
+            print("Here are the remaining customers......")
+            print(retro_video_store.list_of_customers())
+
+        if choice=='6':
+            print_hashtags()
+            print("Here are all of the videos we have in the store.")
+            for video in retro_video_store.list_of_videos():
+                print(video)
+
+        elif choice=='7':
+            id=input("What is the id of the video you want information about? ")
+            if id.isnumeric():
+                id=int(id)
+                response = retro_video_store.get_video(id=id)
+                print_hashtags()
+                print("Here is the video's information:", response)
```

Comment:
More descriptive confirmations provide the best user experience. So if something has been deleted.. mention it :D ! 

```suggestion
                print("Here is the deleted video's information:", response)
```